vagrant_setup.py

```diff
@@ -119,9 +120,11 @@ def check_cli(name, cli_string, msg=None):
 
     vb_usb_filter = f"""vb.customize ["usbfilter", "add", "0",
         "--target", :id,
-        "--name", "{adapter_info['product']} ({adapter_info['manufacturer']})",
-        "--product", "{adapter_info['product']}",
-        "--manufacturer", "{adapter_info['manufacturer']}",
```

Comment:
On my Windows test machine, I had to specifically delete the manufacturer name from the USB filter to get it to match. (Per [this VirtualBox forum thread](https://forums.virtualbox.org/viewtopic.php?f=35&t=82639), the product id and vendor id are still there and should be sufficient.) I think the mismatch was something dumb like "Intel Corp." versus "Intel Corporation" versus "Intel" versus "Intel(R)".